src/main/java/net/fabricmc/loom/task/service/TinyRemapperService.java

```diff
@@ -87,7 +87,7 @@ public static synchronized TinyRemapperService getOrCreate(SharedServiceManager
 				mappings.add(gradleMixinMappingProvider(serviceManager, project.getGradle(), extension.getMappingConfiguration().mappingsIdentifier, from, to));
```

Comment:
(I cannot put a comment on the lines above, but the StringJoiner on line 68 needs to contain either these entries, or some sort of hash, so if multiple projects have diffrent values they will get their own remapper.